### PR TITLE
add URL argument 

### PR DESCRIPTION
**Pull request by nadaa**
Update cli.py and query.py as described here #16  
Allow users to specify a URL with the recipe from the CLI  as 
```
adr <recipe> --url <url>
```

### CODE REVIEW

File: adr/query.py [hunk 1]
```diff
@@ -16,8 +16,12 @@
 from adr.cli import log
 
 here = os.path.abspath(os.path.dirname(__file__))
-
-ACTIVE_DATA_URL = 'http://activedata.allizom.org/query'
+if len(sys.argv)>2:
```
Comment (ahal) [inline adr/query.py]: You won't want to re-parse the raw sys.argv arguments (that's what the ArugmentParser object already did for us). Instead you'll need to use the `args.url` value that you'll get back in `cli.py`.

From there, we need to figure out a good way of propagating that value to `query.py`. One method might be to create a `set_active_data_url` function in `query.py` which changes the value of the global variable. We could call it after parsing the arguments.

Comment (nadaa) [reply adr/query.py]: Thanks for your comments. I 've learned a lot :). I will check it!

File: adr/cli.py [hunk 2]
```diff
@@ -108,6 +106,7 @@ def run_recipe(recipe, args, fmt='table'):
 
 
 def cli(args=sys.argv[1:]):
+    from adr.query import set_active_data_url
```
Comment (ahal) [inline adr/cli.py]: nit: Please move this to the top (under the `adr.formatter` import)

Comment (nadaa) [reply adr/cli.py]: Thanks. I deliberately put the import inside the function, otherwise, a circular import will be raised. What shall I do?

Comment (ahal) [reply adr/cli.py]: Oh, good catch! In that case I think your solution of just putting it in the function is good for now. I'll try and refactor things a bit such that the circular import doesn't happen in the future, but it isn't really related to this issue.

**Comment by ahal** [comment]
Congrats and thanks for you contribution! Please let me know if you'd like to keep working on something else, I'm going to spend a bit of time filing more issues for contributors to pick up today.

**Comment by nadaa** [comment]
Thanks. I will be always happy to help!

**Pull request merged**